dogowner/templates/dogowner/dog_owner_homepage.html

```diff
@@ -0,0 +1,26 @@
+{% extends "main/base_template.html" %}
+{% load static %}
+
+{% block stylesheets %}
+    <link rel="stylesheet" href="{% static 'CSS/dog_owner_homepage.css' %}">
+{% endblock %}
+
+{% block content %}
+<div class="row row-cols-1 g-3 cards">
+  <div style="display: flex; flex-direction: row; flex-wrap: wrap">
+    {% for daycare in daycares %}
+    <div class="card">
+      <img src="{{ daycare.get_daycare_primary_image }}" alt="{{ daycare.name }} image" class="card-img-top">
```

Comment:
In the daycare model, you added the method: `get_daycare_primary_image` that could return null, 
so in that case, you are not referring to that case in the html file.
optional solution: add a field 'daycare_primary_image' in 'dogowner/views.py' in the context array, and there check if is a null, or check it in the html file

Reply:
You're right, but the total review rating of a daycare does not calculated in Hotails ATM.
When we'll implement this method, we'll change the order of presented daycares to the dog owner on his homepage and/or on searching.

Reply:
You must deal with the case you get a None as the image. You can place a placeholder "no image available" or something, but not a broken link.

Reply:
I've added on #77 2 default profile pictures, 1 for daycare and 1 for dog owner.
I'll add it on this PR as well and I'll ping you.